Dockerfile

```diff
@@ -1,16 +1,25 @@
-FROM python:3.7-alpine AS build
+FROM python:3.7-alpine AS base
```

Comment:
let's use 3.8, while we're at it

Reply:
Python-request does not support 3.8 officially.

Reply:
yes. It does. https://github.com/python-gitlab/python-gitlab/commit/6048175ef2c21fda298754e9b07515b0a56d66bd